### PR TITLE
Create en.md

### PR DESCRIPTION
**Pull request by philipecosia**
The English version of the privacy policy

### CODE REVIEW

**Review by AndreGCRamos — APPROVED** [review]

File: en.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+
+## 1. Contact & data protection officer
+In relation to the General Data Protection Regulation (DSGVO), the organisation responsible is: 
+
+Ecosia GmbH  
+Schinkestraße 9  
+12047 Berlin Germany  
+[info@ecosia.org](mailto:info@ecosia.org)
+
+Our Data Protection Officer can be contacted at privacy@ecosia.org or our postal address with the addition of “data protection”.
```
Comment (LostCrew) [inline en.md:12]: In the german version, we used the lower double quote - higher double symbols pair. Is it expected to be different in english?

File: en.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+ - Difference between time zone and Greenwich Mean Time (GMT)
+ - Content of the request (specific webpage)
+ - Access status / HTTP status code
+ - Respectively transferred data quantity
+ - Website from which the request comes
+ - Browser
+ - Operating system and its surface
+ - Language and version of the browser software.
+
+The data are saved in the logfiles of our system. These data and other personal data of the user are not saved. The data are deleted once they are no longer required to achieve the purpose of your gathering. In the event that the data are saved in log files, this occurs at the most seven days later.
```
Comment (LostCrew) [inline en.md:42]: > The data are saved in the logfiles of our system. These data and other personal data of the user are not saved.

The 2 sentences seem to have opposing meanings. Can we verify correctness?

File: en.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+ - Access status / HTTP status code
+ - Respectively transferred data quantity
+ - Website from which the request comes
+ - Browser
+ - Operating system and its surface
+ - Language and version of the browser software.
+
+The data are saved in the logfiles of our system. These data and other personal data of the user are not saved. The data are deleted once they are no longer required to achieve the purpose of your gathering. In the event that the data are saved in log files, this occurs at the most seven days later. 
+
+It is possible to save the data for longer than this. In this case, the users’ IP addresses are deleted or anonymised so that they can no longer be associated with the visiting client.
```
Comment (LostCrew) [inline en.md:44]: Also this is confusing: if we don't store any personal data of the user, we shouldn't be in possess of a data representation of the "visiting client"

File: en.md [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+
+In addition, while you are using our website, cookies are saved on your computer. See below and section 4 in this regard.
+
+We gather anonymous data in order to identify and gather errors and problems in Ecosia products and services. These data include data in connection with program crashes. We use several third-party providers for the gathering of error logs. During the use of the Ecosia’s website, various pieces of personal data that are necessary for the performance are saved on the technical infrastructure of the third-party supplier a limited period of time in encrypted form.
```
Comment (LostCrew) [inline en.md:48]: It might be better to specify how long a "limited period of time" is

File: en.md [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+
+It is possible to save the data for longer than this. In this case, the users’ IP addresses are deleted or anonymised so that they can no longer be associated with the visiting client.
+
+In addition, while you are using our website, cookies are saved on your computer. See below and section 4 in this regard.
+
+We gather anonymous data in order to identify and gather errors and problems in Ecosia products and services. These data include data in connection with program crashes. We use several third-party providers for the gathering of error logs. During the use of the Ecosia’s website, various pieces of personal data that are necessary for the performance are saved on the technical infrastructure of the third-party supplier a limited period of time in encrypted form. 
+
+## 5. Cookies
+
+Cookies are small text files which are saved on your hard drive assigned to the browser you have used and through which the location which the cookie sets (here by us) accrues specific pieces of information.   Cookies cannot execute any programs or transmit viruses to your computer. They serve to make Ecosia more user-friendly and more effective in general.
```
Comment (LostCrew) [inline en.md:52]: `   ` multiple spaces before `Cookies`?

File: en.md [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+
+We set one or several persistent cookies in order to save the user’s settings and to make our website more user-friendly, to analyse user behaviour and as necessary carry out web analytics (see below in this regard) or A/B testing (see below also). The legal basis for the processing of personal data using cookies is Art. 6 Para. 1 lit. f of the General Data Protection Regulation.
```
Comment (LostCrew) [inline en.md:54]: `make our website more user-friendly` is very generic and also repeated twice

File: en.md [hunk 7]
```diff
@@ -0,0 +1,230 @@
...
+ - Your IP address 
+ - At the same time, we use cookies to gather data for statistical purposes and to avoid misuse. 
+
+The processing of your personal data for the use of our enquiry and contact function is carried out on the basis of your consent.
+
+The headquarters of Zendesk are located in the US, and your data, gathered when using our enquiry and contact function, are sent there. Zendesk is certified under the “Privacy Shield” US-EU data protection agreement and undertakes to uphold the EU data protection provisions. https://www.privacyshield.gov/participant?id=a2zt0000000TO6hAAG
+
+### g. Web analytics
+
+We use a safe and fully encrypted web analysis service, which we ourselves operate, in order to analyse and continuously improve the use of our website. Using the statistics obtained, we can improve our offering and design it in a more interesting way for you as a user. The legal basis for this processing of your personal data is Art. 6 Abs. 1 lit. f of the General Data Protection Regulation.
```
Comment (LostCrew) [inline en.md:148]: We could be more explicit about the extent of the encryption

**Pull request merged**